src/data/scripts/hullmods/VayraDamagedShields.java

```diff
@@ -6,14 +6,29 @@
 import com.fs.starfarer.api.combat.ShipAPI.HullSize;
 import com.fs.starfarer.api.impl.campaign.ids.Stats;
 import com.fs.starfarer.api.impl.hullmods.CompromisedStructure;
+import data.scripts.util.MiscUtils;
+import data.util.LoggerLogLevel;
 
 public class VayraDamagedShields extends BaseHullMod {
+    private static final String LOGTAG = "VayraDamagedShields";
+    public static boolean DISABLE_FOR_PLAYER = false;
+    public static boolean DISABLE_FOR_ENEMY = false;
 
     public static final float SHIELD_UPKEEP_MULT = 1.5f;
     public static final float SHIELD_DAMAGE_MULT = 1.1f;
 
     @Override
     public void applyEffectsBeforeShipCreation(HullSize hullSize, MutableShipStatsAPI stats, String id) {
+        // Do nothing if it's been disabled
+        if (stats.getFleetMember() != null) {
+            int owner = stats.getFleetMember().getOwner();
+            if (owner == MiscUtils.OWNER_ENEMY && DISABLE_FOR_ENEMY) return;
+            if (owner == MiscUtils.OWNER_PLAYER && DISABLE_FOR_PLAYER) return;
```

Comment:
ditto

Reply:
in fact, change this to 
```
if (DISABLE_FOR_ENEMY && MiscUtils.OWNER_ENEMY == stats.getFleetMember.getOwner()) return;
if (DISABLE_FOR_PLAYER && MiscUtils.OWNER_PLAYER == stats.getFleetMember.getOwner()) return;
```

but that's just a nitpick; reversing the order is kind of a much bigger deal.

Reply:
done. resolving.